### PR TITLE
Pull Request for Issue1027: Finish time based acquisitions

### PR DESCRIPTION
**Pull request by dretrex**
I have finished the time based scan acquisitions.  It operates in a ```<acquisition time> every <overall time period> <N> times```.  I'm sure there are other things that could be added, but as it stands now, it is finished to it's original spec.

### CODE REVIEW

File: source/acquaman/AMTimedScanActionControllerAssembler.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+AMTimedScanActionControllerAssembler::~AMTimedScanActionControllerAssembler()
+{
+
+}
+
+bool AMTimedScanActionControllerAssembler::generateActionTreeImplmentation()
+{
+	AMLoopAction3 *actionTree = new AMLoopAction3(iterations_);
```
Comment (davidChevrier) [inline source/acquaman/AMTimedScanActionControllerAssembler.cpp]: Naming is a little confusing here ... but I figured it out in the end

File: source/acquaman/AMTimedScanActionControllerAssembler.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	actionTree_ = scanActions;
+
+	return true;
+}
+
+AMAction3* AMTimedScanActionControllerAssembler::generateActionListForDetectorAcquisition()
+{
+	AMListAction3 *retVal = new AMListAction3(new AMListActionInfo3(QString("Acquire All Detectors"), QString("Acquire %1 Detectors").arg(detectors_->count())), AMListAction3::Parallel);
+	qDebug() << detectors_->count();
```
Comment (davidChevrier) [inline source/acquaman/AMTimedScanActionControllerAssembler.cpp]: qDebug can be removed?

File: source/acquaman/AMTimedScanActionControllerAssembler.cpp (hunk 1 above)
Comment (dretrex) [inline source/acquaman/AMTimedScanActionControllerAssembler.cpp]: If you have a better set of definitions I'll switch them.  They seemed the most appropriate at the time.

File: source/acquaman/AMTimedScanActionControllerAssembler.cpp (hunk 2 above)
Comment (dretrex) [inline source/acquaman/AMTimedScanActionControllerAssembler.cpp]: Yeah, probably.

File: source/acquaman/AMTimedScanActionControllerAssembler.cpp (hunk 1 above)
Comment (davidChevrier) [inline source/acquaman/AMTimedScanActionControllerAssembler.cpp]: Just that the "value loop" is called actionTree. It's a subaction of scanActions. And the actionTree_ member variable is set to scanActions. Maybe even renaming the first one mainLoop or something. Really, not that big a deal.

**Pull request merged**